SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -106,7 +106,9 @@ USpatialGDKSettings::USpatialGDKSettings(const FObjectInitializer& ObjectInitial
 	, bBatchSpatialPositionUpdates(false)
 	, MaxDynamicallyAttachedSubobjectsPerClass(3)
 	, ServicesRegion(EServicesRegion::Default)
-	, WorkerLogLevel(ESettingsWorkerLogVerbosity::Warning)
+	, WorkerLogLevel(ESettingsWorkerLogVerbosity::Warning) // Deprecated - UNR-4348
+	, LocalWorkerLogLevel(WorkerLogLevel)
```

Comment:
Hmm, I think this one may be too early - as in the two WorkerLogLevels will always be set to Warning, since the WorkerLogLevel will first be set to Warning, and then may later be overriden by the config settings. But worth testing by changing the WorkerLogLevel in the config, and then printing the value of the LocalWorkerLogLevel and CloudWorkerLogLevel.

Reply:
Tested!

Reply:
So, just to clarify, if the config had a different value, e.g. Error, we would want to initialize the new log levels also to Error (@mironec I assume this is what you meant?). Is this currently going to be the behavior?

Reply:
So, if the config had a different value, e.g. Error, then the WorkerLogLevel will first be set to Warning, then to Error after reading from config, and the two new settings LocalWorkerLogLevel and CloudWorkerLogLevel will be set to Error in this case too